app/routes/pages.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    return this.l10n.t('Pages');
+  },
+  model(params) {
+    let externalPage;
+    this.modelFor('application').pages.forEach(page => {
```

Comment:
Please use find instead of for each

Reply:
@niranjan94 Here you meant, ember data find or lodash find?